server/src/stateManagement.mjs

```diff
@@ -584,13 +583,12 @@ function updateState(userId, newState, scope = "") {
   }
   //to get the userId from given user/appId
   scope = getUserId(scope);
-
   if ( scope in state ){
-    userState = getState(scope);
+    userState = getState(scope,false);
```

Comment:
It seems strange to use 'scope' as a variable name that holds an actual user id (line 581). I think of 'scope' as either the literal string 'global' or the literal string 'group' or the literal string 'user' or something like that.